server/commands/check.go

```diff
@@ -110,16 +108,40 @@ func (query *CheckQuery) Execute(ctx context.Context, req *openfgapb.CheckReques
 }
 
 // getTypeDefinitionRelationUsersets validates a tuple and returns the userset corresponding to the "object" and "relation"
-func (query *CheckQuery) getTypeDefinitionRelationUsersets(ctx context.Context, rc *resolutionContext) (*openfgapb.Userset, error) {
-	ctx, span := query.tracer.Start(ctx, "getTypeDefinitionRelationUsersets")
-	defer span.End()
+func (query *CheckQuery) getTypeDefinitionRelationUsersets(tk *openfgapb.TupleKey, typesys *typesystem.TypeSystem) (*openfgapb.Userset, error) {
```

Comment:
Does this need to be a method on `*CheckQuery`?

Reply:
Nope! It's a static method for sure. Good call. Changed it.